src/server/scripts/EasternKingdoms/Deadmines/instance_deadmines.cpp

```diff
@@ -70,10 +66,12 @@ class instance_deadmines : public InstanceMapScript
             ObjectGuid DefiasPirate1GUID;
             ObjectGuid DefiasPirate2GUID;
             ObjectGuid DefiasCompanionGUID;
+            ObjectGuid MrSmiteGUID;
 
             uint32 State;
             uint32 CannonBlast_Timer;
             uint32 PiratesDelay_Timer;
+            uint32 SmiteAlarmDelay_Timer;
```

Comment:
this field is not initialized in the ctor, you should always make sure all fields are initialized